src/test/java/APICalls/RetrofitCall.java

```diff
@@ -0,0 +1,80 @@
+//This file is failed attempt to make a working call to Trelo API using Retrofit
+//After multiple attempts and making a successful call to other API in SimpleRetrofitForChecking Package
+//I believe there is something wrong in the response of Trelo API
+//Whole file is a mess because I don't see a point in fixing and reformatting code that doesn't even work
+
+package APICalls;
+
+import RetrofitModels.ResponseModel;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+import retrofit2.converter.gson.GsonConverterFactory;
+import retrofit2.http.*;
+
+
+import java.io.IOException;
+
+public class RetrofitCall extends BaseCall {
+    protected static String POST_URL = "https://api.trello.com/1/boards/?name=test_board&key="+ apiKey+"&token="+ apiToken;
+
+    static String URL = "https://api.trello.com/1/boards/?name=test_board&key=d1290b394d0d3023ee548174c8fb3d85&token=ATTAb18cc18128f5aa873a3f6158fde484c048479039858026047b2feaa21ba56846B0D5784C";
+    interface ApiInterface{
+        @GET("/")
+        Call<ResponseModel> getBoard(@Query("id") String id, @Query("apiKey") String apiKey, @Query("apiToken") String apiToken );
+
+        @POST("/?name=test_board")
+        Call<ResponseModel> postBoard(@Query("apiKey") String apiKey, @Query("apiToken") String apiToken );}
+
+     class ApiClient {
```

Comment:
you can also implement a method to create and configure a Retrofit client, for example:
```
    private static Retrofit buildRetrofitClient() {
        return new Retrofit.Builder()
                .baseUrl("https://api.trello.com/1/")
                .addConverterFactory(GsonConverterFactory.create())
                .build();
    }
```